src/include/storage/block_access_controller.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+#include <atomic>
+#include <utility>
+#include "common/macros.h"
+#include "common/strong_typedef.h"
+
+namespace terrier::storage {
+/**
+ * TODO(Tianyu): Placeholder for the actual implementation that has the same number of bytes. We use this to make sure
```

Comment:
Can you give a brief description of what this will be, since it's going to confuse people working on course projects to see this in the codebase?

Reply:
Good point. Copied the docs from actual blocks.